protractor.conf.js

```diff
@@ -2,7 +2,7 @@
 // https://github.com/angular/protractor/blob/master/docs/referenceConf.js
 
 /*global jasmine */
-var SpecReporter = require('jasmine-spec-reporter');
+var SpecReporter = require('jasmine-spec-reporter').SpecReporter;
```

Comment:
We require es6/es2015 (whatever it's called nowadays) for build time at least, so prefer `let`.